CrowdStrike/crowdstrike_falcon/ingest/parser.yml

```diff
@@ -114,6 +114,8 @@ stages:
           process.name: "{{parsed_event.message.event.FileName}}"
           process.working_directory: "{{parsed_event.message.event.FilePath}}"
           crowdstrike.detect_id: "{{parsed_event.message.event.DetectId}}"
+          crowdstrike.ioc.type: "{{parsed_event.message.event.IOCType}}"
+          crowdstrike.ioc.value: "{{parsed_event.message.event.IOCValue}}"
```

Comment:
IMHO, [`threat.indicator.type`](https://www.elastic.co/guide/en/ecs/8.17/ecs-threat.html#field-threat-indicator-type) and [`threat.indicator.name`](https://www.elastic.co/guide/en/ecs/8.17/ecs-threat.html#field-threat-indicator-name) are more suitable.
For `threat.indicator.type`, as the value are not the same as ones in `IOCType`, you have to map them:
- ECS values: https://www.elastic.co/guide/en/ecs/8.17/ecs-threat.html#field-threat-enrichments-indicator-type
- IOCType Values: domain, filename, hash_md5, hash_sha256, registry_key, command_line, behavior